modules/S16_ghidra_decompile_checks.sh

```diff
@@ -24,13 +24,13 @@ S16_ghidra_decompile_checks()
   pre_module_reporter "${FUNCNAME[0]}"
 
   if [[ ${BINARY_EXTENDED} -ne 1 ]] ; then
-    print_output "[-] ${FUNCNAME[0]} - missing ghidra_scripts dependencies"
+    print_output "[-] ${FUNCNAME[0]} - missing ghidra_scripts dependencies, no ${BINARY_EXTENDED}"
```

Comment:
You are right, the naming is not great ;)
BINARY_EXTENDED is for enabling/disabling the Ghidra checks. Probably we should adjust this to
```
print_output "...<snip>..., BINARY_EXTENDED not set to 1. You can set it up via a scan-profile."

Reply:
btw with a current docker-image you should not get this error at all. The docker image should get checked in the initial dependency checker.

Reply:
Ok, sure.
Btw. an update.sh next to install.sh would be great, I forgot to do the docker pull....

```
$ cat update.sh                                                                                                                 
git pull
docker pull embeddedanalyzer/emba
cd external/nvd-json-data-feeds
sudo git pull
```
of course maybe some error checks to figure out if nvd was installed with sudo or not would be good